pkg/mapper/dynamicfile/dynamicfile.go

```diff
@@ -21,9 +25,25 @@ type DynamicFileMapStore struct {
 	filename                  string
 	userIDStrict              bool
 	usernamePrefixReserveList []string
+
+	skipFirstTimeMetric bool
+}
+
+// Meta is the collection of fields which should be included in all top-level
+// objects which are propagated for dynamic file mode
+type Meta struct {
+	APIVersion string `json:"ApiVersion"`
```

Comment:
i'd remove APIVersion if its not used. If this field is not persisted, i can't imagine what it will be used for